### PR TITLE
feat: pdf viewer

### LINKED ISSUE #28: PDF Flip Book
Original site uses[ PDF Flip Book plugin](https://dearflip.com/realistic-3d-flipbook-wordpress-plugin/) to showcase product catalog: https://www.mammotome.com/us-product-catalog/

Customer query pending whether exact same plugin is expected or alternatives can be researched. Issue will be updated.

Similar use case implemented for https://main--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/agratec-news-als-e-paper-zum-durchblaettern

### PR DESCRIPTION
**Pull request by solaris007**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #28 

Test URLs:
- Before: https://main--mammotome--hlxsites.hlx.page/en/dj-test-pdf-viewer
- After: https://28-pdf-flip-book--mammotome--hlxsites.hlx.page/en/dj-test-pdf-viewer

### CODE REVIEW

File: blocks/pdf-viewer/pdf-viewer.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    loadScript(ADOBE_DC_VIEW_SDK_SRC, onEmbedPDFScriptLoaded);
+  }
+};
+
+const addEventListeners = (config) => {
+  document.addEventListener(
+    ADOBE_DC_VIEW_SDK_READY_EVENT,
+    createAdobeDCViewSDKReadyHandler(config),
+  );
+  document.addEventListener(
```
Comment (solaris007) [inline blocks/pdf-viewer/pdf-viewer.js:79]: currently viewer is loaded as part of delayed. we may need to add better styling / placeholder or even change behavior on-click as a followup.

File: blocks/pdf-viewer/pdf-viewer.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
+import { fetchPlaceholders, loadScript } from '../../scripts/lib-franklin.js';
+
+/**
+ * https://developer.adobe.com/document-services/docs/overview/pdf-embed-api/gettingstarted/
+ */
+const ADOBE_DC_VIEW_SDK_SRC = 'https://documentservices.adobe.com/view-sdk/viewer.js';
+const ADOBE_DC_VIEW_SDK_READY_EVENT = 'adobe_dc_view_sdk.ready';
+const FRANKLIN_DELAYED_COMPLETED_EVENT = 'franklin.delayed_completed';
+
+const siteConfig = (await fetchPlaceholders()).config || {};
```
[the remark this answers is not in the document]
Comment (solaris007) [reply blocks/pdf-viewer/pdf-viewer.js:10]: yes, i will do that, thank you

**Comment by solaris007** [comment]
@Acidham this is a general issue with section styling and affects all blocks. i will open a follow-up issue to fix it for all.

**Comment by solaris007** [comment]
@karlpauls PTAL

**Review by Acidham — COMMENTED** [review]
align center

**Review by karlpauls — APPROVED** [review]

**Pull request merged**